evap/staff/tests/test_views.py

```diff
@@ -272,13 +272,25 @@ def setUpTestData(cls):
         cls.testuser = baker.make(UserProfile)
         cls.url = f"/staff/user/{cls.testuser.pk}/edit"
 
-    def test_questionnaire_edit(self):
+    def test_user_edit(self):
         page = self.app.get(self.url, user=self.manager, status=200)
         form = page.forms["user-form"]
         form["email"] = "lfo9e7bmxp1xi@institution.example.com"
         form.submit()
         self.assertTrue(UserProfile.objects.filter(email="lfo9e7bmxp1xi@institution.example.com").exists())
 
+    def test_user_edit_duplicate_email(self):
+        second_user = baker.make(UserProfile)
+        second_user.email = "lfo9e7bmxp1xi@institution.example.com"
+        second_user.save()
```

Comment:
```suggestion
        second_user = baker.make(UserProfile, email="lfo9e7bmxp1xi@institution.example.com")
```

Reply:
(we could also use a less cryptic looking mailbox-part than "lfo9e7bmxp1xi" here. I don't know why the existing test used that, may have been about accidental collisions, but that shouldn't be a problem anymore)